Dockerfile.assisted-installer-build

```diff
@@ -1,19 +1,34 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.19
+FROM registry.access.redhat.com/ubi9/go-toolset:1.18 AS golang
 
-USER 0
-
-ENV GO111MODULE=on
 ENV GOFLAGS=""
-ENV GOPATH="/go"
-ENV PATH="$PATH:$GOPATH/bin"
 
-RUN go install golang.org/x/tools/cmd/goimports@v0.1.0 && \
-    go install github.com/onsi/ginkgo/ginkgo@v1.16.1 && \
-    go install github.com/golang/mock/mockgen@v1.6.0 && \
-    go install gotest.tools/gotestsum@v1.6.3 && \
-    go install github.com/axw/gocov/gocov@latest && \
-    go install github.com/AlekSi/gocov-xml@latest
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.46.0 && \ 
+  go install golang.org/x/tools/cmd/goimports@v0.1.0 && \
+  go install github.com/onsi/ginkgo/ginkgo@v1.16.1 && \
+  go install github.com/golang/mock/mockgen@v1.6.0 && \
+  go install gotest.tools/gotestsum@v1.6.3 && \
+  go install github.com/axw/gocov/gocov@latest && \
+  go install github.com/AlekSi/gocov-xml@latest
+
+FROM quay.io/centos/centos:stream9
+
+RUN dnf install -y 'dnf-command(config-manager)' && \
+    dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo && \
+    dnf install -y \
+        docker-ce \
+        docker-ce-cli \
+        containerd.io \
+        make \
+        git \
+        openssl-devel \
+        gcc \
```

Comment:
I would also install util-linux explicitly. I think it comes with one of the docker stuff, but without it I had trouble to runner skipper locally

Reply:
why would util-linux inside the container would relate to skipper on the host ? I don't need for skipper to work

Reply:
https://github.com/Stratoscale/skipper/blob/upstream/skipper/data/skipper-entrypoint.sh#L34

Reply:
Oh this is because you are using later version of skipper, I use 2.0.2

Reply:
OK lets add it for any case

Reply:
I think this problem would occur for you with a lot of other Dockerfiles we use. Maybe add it in different PRs ?

Reply:
Although I would recommend using skipper==2.0.2